pkg/prometheus/rules_test.go

```diff
@@ -160,10 +161,33 @@ func shouldSplitUpLargeSmallIntoTwo(t *testing.T) {
 	}
 
 	if len(configMaps) != 2 {
-		t.Fatalf("expected rule files to be split up into two ConfigMaps, but got '%v' instead", len(configMaps))
+		t.Fatalf("expected rule files to be split up into three ConfigMaps, but got '%v' instead", len(configMaps))
 	}
 
 	if configMaps[0].Data["first"] != ruleFiles["first"] || configMaps[1].Data["second"] != ruleFiles["second"] {
 		t.Fatal("expected ConfigMap data to match rule file content")
 	}
 }
+
+func shouldSplitUpLargeSmallIntoFour(t *testing.T) {
+	p := &monitoringv1.Prometheus{}
+	ruleFiles := map[string]string{}
+
+	ruleFiles["first"] = strings.Repeat("a", maxConfigMapDataSize)
+	ruleFiles["second"] = strings.Repeat("a", maxConfigMapDataSize)
+	ruleFiles["three"] = strings.Repeat("a", maxConfigMapDataSize)
+	ruleFiles["four"] = "a"
+
+	configMaps, err := makeRulesConfigMaps(p, ruleFiles)
```

Comment:
Since here we are testing only the function `makeRulesConfigMaps` I don't think adding this test makes sense as it's testing the same behaviour as `shouldSplitUpLargeSmallIntoTwo`. Instead, it might be better to test `createOrUpdateRuleConfigMaps` since that's the function that we actually changed the behaviour